server/channels/app/channel.go

```diff
@@ -1568,6 +1568,12 @@ func (a *App) addUserToChannel(c request.CTX, user *model.User, channel *model.C
 		return nil, model.NewAppError("AddUserToChannel", "app.channel_member_history.log_join_event.internal_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 
+	if user.IsGuest() {
+		go func() {
+			a.Srv().telemetryService.SendTelemetryForFeature("guest", false, "add_guest_to_channel", map[string]any{})
```

Comment:
Thinking out loud on this one.

- Does it make sense to use an enum holding the name of the features? It should help avoid typing errors, while it will also help having a list of tracked features.
- Shouldn't the id of the user inviting the guest to the channel be a part of the properties? `user_actual_id` is reserved for this purpose.

Reply:
this is looking into tracking guest users. If there is value in tracking any user, I can modify it to make it work, but it was not the original purpose (I'm 0/5 on it, though)